<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>277</width>
    <height>313</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QVBoxLayout" name="verticalLayout_2">
    <item>
     <widget class="QGroupBox" name="groupBox">
      <property name="title">
       <string>GroupBox</string>
      </property>
      <layout class="QVBoxLayout" name="verticalLayout">
       <item>
        <widget class="QCheckBox" name="argsCheckBox">
         <property name="text">
          <string>args</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QCheckBox" name="cmdCheckBox">
         <property name="text">
          <string>cmd</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QCheckBox" name="commCheckBox">
         <property name="text">
          <string>comm</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QCheckBox" name="pidCheckBox">
         <property name="text">
          <string>pid</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QCheckBox" name="ppidCheckBox">
         <property name="text">
          <string>ppid</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QCheckBox" name="ttyCheckBox">
         <property name="text">
          <string>tty</string>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </item>
    <item>
     <spacer name="verticalSpacer">
      <property name="orientation">
       <enum>Qt::Orientation::Vertical</enum>
      </property>
      <property name="sizeHint" stdset="0">
       <size>
        <width>20</width>
        <height>40</height>
       </size>
      </property>
     </spacer>
    </item>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout">
      <item>
       <widget class="QPushButton" name="cancelButton">
        <property name="text">
         <string>Cancel</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="okButton">
        <property name="text">
         <string>Ok</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>277</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
